.travis.yml

```diff
@@ -6,9 +6,9 @@ cache:
 notifications:
   email: false
 node_js:
+  - '7'
   - '6'
   - '4'
-  - '0.12'
```

Comment:
mocha-typescript uses `let` and possibly other es2015 features not present in `0.12`, so I figured it was time to let support for that version go.